.eslintrc.js

```diff
@@ -62,12 +65,12 @@ module.exports = {
     "new-parens": "error",
     "no-caller": "error",
     "no-bitwise": "off",
-    "no-conditional-assignment": true,
```

Comment:
why did you remove this rule :(

Reply:
I'll add it back. It was giving me weird issues

Reply:
First time messing with the the eslint config 😅